entry_types/scrolled/package/src/frontend/index.js

```diff
@@ -14,6 +14,8 @@ const editMode = window.location.pathname.indexOf('/editor/entries') === 0;
 export {api as frontend} from './api';
 
 export * from './Image';
+export * from './useImageUrl';
+export * from './useImageAlt';
```

Comment:
These two need are only needed until `useFile` is fixed for usage in content elements. But for now they can stay here.